app/src/main/java/com/android/voyageur/model/user/UserViewModel.kt

```diff
@@ -128,11 +136,19 @@ open class UserViewModel(
    * @param userId The ID of the user to add as a contact.
    */
   fun addContact(userId: String) {
-    val contacts = user.value?.contacts?.toMutableSet()
-    val newUser = user.value!!.copy()
-    contacts?.add(userId)
-    newUser.contacts = contacts?.toList().orEmpty()
-    if (user.value != null) updateUser(newUser)
+    //    val contacts = user.value?.contacts?.toMutableSet()
+    //    val newUser = user.value!!.copy()
```

Comment:
Nit: Here removing the commented lines would improve the readability. I leave it up to you to remove them.

Reply:
Will remove this and merge the PR. Thanks for the review!